osidb/helpers.py

```diff
@@ -17,6 +18,18 @@
 from .exceptions import OSIDBException
 
 
+def cve_id_comparator(cve_id):
+    """
+    comparator to sort CVE IDs by
+
+        1) the year
+        2) the sequence
+    """
+    digits = re.sub(r"[^0-9]", "", cve_id)
+    # stress the value of the year above the sequence
+    return int(digits[:4]) ** 2 + int(digits[4:])
```

Comment:
@Elkasitu one example is `2020-11111` and `2020-9999` and there is obviously many of them. I am opened to suggestions but this should otherwise work OK. Of course that the string comparison would always handle the year part correctly but I did not want to compare both parts separately in two steps. This way both the year and the order values may be just easily joined and form a single comparable value.